packages/fxa-payments-server/src/store/state.ts

```diff
@@ -12,14 +12,19 @@ import {
   UpdateSubscriptionPlanResult,
 } from './types';
 
+import { actions, ActionPayload } from './actions';
+
 export const defaultState = {
   customer: uninitializedFetch<Customer, APIError>(),
   plans: uninitializedFetch<Array<Plan>, APIError>(),
   profile: uninitializedFetch<Profile, APIError>(),
   subscriptions: uninitializedFetch<Array<Subscription>>(),
   token: uninitializedFetch<Token>(),
   cancelSubscription: uninitializedFetch<Subscription>(),
-  reactivateSubscription: uninitializedFetch<any>(),
+  reactivateSubscription: uninitializedFetch<
+    ActionPayload<typeof actions['reactivateSubscription']>,
```

Comment:
This is the first bit using `ActionPayload` - plumbing for the response type of the reactivate subscription API through the Redux store.